### ISSUE TITLE
Bluetooth?

### ISSUE THREAD

**Issue by ghost** [flynn/u2f#2]
After taking a look at the code, I assume that u2ftoken could be used with bluetooth FIDO u2f devices that adhere to the `Device` interface? I am looking to implement u2f over bluetooth for smartwatches (such as the Pebble Time Steel and Android Wear)... would love to use this library rather than write another...

**Comment by titanous** [comment]
Yep, the packages were explicitly designed to be layered so that you can drop a BLE transport in instead of USB HID.

**Comment by ghost** [comment]
Awesome!

-sent from my OnePlus X
